src/search.c

```diff
@@ -309,7 +309,7 @@ static GtkWidget *add_find_checkboxes(GtkDialog *dialog)
 	ui_hookup_widget(dialog, check_regexp, "check_regexp");
 	gtk_button_set_focus_on_click(GTK_BUTTON(check_regexp), FALSE);
 	gtk_widget_set_tooltip_text(check_regexp, _("Use POSIX-like regular expressions. "
-		"For detailed information about using regular expressions, please read the documentation."));
+		"For tips about using regular expressions, please read the Geany manual."));
```

Comment:
and I don't like the end either, because IMO "the documentation" implicitly refers to Geany's just fine, and that we use it everywhere else (I can find).  Also, it feels strange to mention it's Geany's manual in a tooltip from inside Geany…